bsconfig.json

```diff
@@ -15,13 +15,12 @@
   "suffix": ".bs.js",
   "bs-dependencies": [
     "@rescript/react",
-    "bs-jest",
-    "bs-fetch",
-    "@glennsl/bs-json"
+    "@glennsl/rescript-fetch",
+    "@rescript/core"
   ],
-  "reason": {
-    "react-jsx": 3
+  "jsx": {
+    "version": 4
   },
-  "bsc-flags": ["-bs-super-errors"],
+  "bsc-flags": ["-open RescriptCore"],
```

Comment:
We now open `ReScriptCore` by default for every module. This will expose a refined set of "standard library modules", such as `Array`, `List`, `String`, etc.

Note that the original `Array`, `List` etc. have different interfaces, and use the "data last" convention, which forced us to use the `|>` operator to pipe into the last parameter position.

The new apis are essentially pipe-first, so we'll literally move forward with `->` operator instead.

Example:

```
// Old Array.map
[1,2,3] |> Array.map((a) => a + 1)

// New Array.map
[1,2,3]->Array.map((a) => a + 1)
```